cmd/juju/backups/restore.go

```diff
@@ -78,29 +44,18 @@ type RestoreAPI interface {
 }
 
 var restoreDoc = `
-Restores a backup that was previously created with "juju create-backup".
-
-This command creates a new controller and arranges for it to replace
-the previous controller for a model.  It does *not* restore
-an existing server to a previous state, but instead creates a new server
-with equivalent state.  As part of restore, all known instances are
-configured to treat the new controller as their master.
-
-The given constraints will be used to choose the new instance.
+Restores a backup that was previously created with "juju create-backup",
```

Comment:
Restores the Juju controller state database from a backup that was previously created with "juju create-backup".

Reply:
Not sure on this. I feel like it should be clear this only is restoring the DB vs more aspects of Juju and controllers.

Reply:
i'm on the fence with specifically calling out controller.  backup and restore only works if the model is controller, however it doesn't assume the controller model is being used either.  perhaps we're trying leave the door open for later allowing for a different granularity of backups?  or is left over from juju 1.x?

Reply:
Perhaps a paragraph to say that existing units will not be removed, nor will new ones be created to fit the database being restored should be added?  With more detail.

Reply:
The command isn't limited to only folks that are admins and folks may try to run it, get errors, etc. So the help has to be clear enough to everyone. This is why I think being explicit about what it's doing it best. I'm +1 on mentioning that units won't be removed/updated/etc. I do think that saying that restore is only updating the state database is a helpful clarification.